src/db_loader.nit

```diff
@@ -69,7 +69,7 @@ for player in players do
 	# load some statuses
 	for mission in config.missions.find_all do
 		var status = new MissionStatus(mission, player, mission.track)
-		if mission.is_unlocked_for_player(config, player) or 100.rand > 25 then
+		if mission.is_unlocked_for_player(config, player) or 100.rand > 25 or mission.parents.is_empty then
```

Comment:
This check is already done in `is_unlocked_for_player`:

https://github.com/privat/Missions/blob/3daa2d1a0bcd987343c3cc22cc7f3ff9eb4c657b/src/model/status.nit#L66